cmd/dockerd/config.go

```diff
@@ -53,6 +55,8 @@ func installCommonConfigFlags(conf *config.Config, flags *pflag.FlagSet) {
 
 	flags.StringVar(&conf.MetricsAddress, "metrics-addr", "", "Set default address and port to serve the metrics api on")
 
+	flags.Var(opts.NewQuotedString(&conf.TrustKeyPath), "key-path", "Path to key file for ID and image signing")
```

Comment:
Problem is that it is not really used for trust anymore since the signatures are not checked and the ID is not in Swarm mode for cryptographic identity. It is basically just used for deprecated features and not something users will want to set themselves. I only added a flag because the config parser required having the flag as well.